### ISSUE TITLE
Cookie domain

### ISSUE THREAD

**Issue by OpenEyeNet** [signalpoint/angular-drupal#36]
In drupal you can set the cookie domain in your settings.php.

Can you authenticate using that?

**Comment by webdobe** [comment]
Yes.  That when calling drupal.connect(); It will auto authenticate using the cookie_domain.

**Comment by signalpoint** [comment]
@OpenEyeNet and @webdobe Did you need the cookie_domain because your Drupal site and your app are running in different domains? Or did you need cookie_domain even when they are running in the same domain?

Also, are you both working with D7 and/or D8 here? I need to wrangle all these cookie/login/cors/subdomain issues folks are having, and get things under control for both D7 and D8.

**Comment by webdobe** [comment]
So my setup is I have the main domain as the drupal 7 site.  But I have a few different silo apps.  That I want to be able to share users across domains.  This is where the cookie_domain came in.  So if someone logs in at one site.. They will be logged into all of them.  After I got all the CORS stuff straight, I didn't have to modify anything in angular-drupal.   So I have a feeling that all peoples issues stem from CORS.  

I posted what I did @ #https://github.com/easystreet3/angular-drupal/issues/2

I plan on taking the D8 version through a development cycle next so.  I will be able to give feedback on that then.  Really liking angular and headless drupal.
